Makefile

```diff
@@ -1,148 +1,26 @@
-
-image: image-amd64 image-arm64
-
-debug-image-amd64:
-	mkdir -p out/amd64
-	linuxkit build --docker -format kernel+initrd -name debug-x86_64 -dir out hook_debug.yaml
-
-debug-image-arm64:
-	mkdir -p out/arm64
-	linuxkit build --docker -arch arm64 -format kernel+initrd -name debug-aarch64 -dir out hook_debug.yaml
-
-debug-image: debug-image-amd64 debug-image-arm64
-
-run-amd64:
-	sudo ~/go/bin/linuxkit run qemu --mem 2048 out/hook-x86_64
-
-run-arm64:
-	sudo ~/go/bin/linuxkit run qemu --mem 2048 out/hook-aarch64
-
-run:
-	sudo ~/go/bin/linuxkit run qemu --mem 2048 out/hook-${ARCH}
-
-dev-bootkitBuild:
-	cd bootkit; docker buildx build --load -t $(ORG)/hook-bootkit:0.0 .
-
-bootkitBuild:
-	cd bootkit; docker buildx build --platform linux/amd64,linux/arm64 --push -t $(ORG)/hook-bootkit:0.0 .
-
-dev-tink-dockerBuild:
-	cd tink-docker; docker buildx build --load -t $(ORG)/hook-docker:0.0 .
-
-tink-dockerBuild:
-	cd tink-docker; docker buildx build --platform linux/amd64,linux/arm64 --push -t $(ORG)/hook-docker:0.0 .
-
-dev-convert:
-	rm -rf ./convert
-	mkdir ./convert
-	cp out/hook-${ARCH}-initrd.img ./convert/initrd.gz
-	cd convert/; gunzip ./initrd.gz; cpio -idv < initrd; rm initrd; find . -print0 | cpio --null -ov --format=newc > ../initramfs-${ARCH}; gzip ../initramfs-${ARCH}
-
-.PHONY: convert
-convert:
-	for a in x86_64 aarch64; do \
-		rm -rf ./convert; \
-		mkdir ./convert; \
-		cp out/hook-$$a-initrd.img ./convert/initrd.gz; \
-		cd convert/; gunzip ./initrd.gz; cpio -idv < initrd; rm initrd; find . -print0 | cpio --null -ov --format=newc > ../initramfs-$$a; gzip ../initramfs-$$a; cd ../;\
-	done
-
-dist: default convert
-	rm -rf ./dist ./convert
-	mkdir ./dist
-	for a in x86_64 aarch64; do \
-		mv ./initramfs-$$a.gz ./dist/initramfs-$$a; \
-		mv ./out/hook-$$a-kernel ./dist/vmlinuz-$$a; \
-	done
-	rm -rf out
-	cd ./dist && tar -czvf ../hook-${GIT_VERSION}.tar.gz ./*
-
-dist-existing-images: image convert
-	rm -rf ./dist ./convert
-	mkdir ./dist
-	for a in x86_64 aarch64; do \
-		mv ./initramfs-$$a.gz ./dist/initramfs-$$a; \
-		mv ./out/hook-$$a-kernel ./dist/vmlinuz-$$a; \
-	done
-	rm -rf out
-	cd ./dist && tar -czvf ../hook-${GIT_VERSION}.tar.gz ./*
-
-
-dev-dist: dev dev-convert
-	rm -rf ./dist ./convert
-	mkdir ./dist
-	mv ./initramfs-${ARCH}.gz ./dist/initramfs-${ARCH}
-	mv ./out/hook-${ARCH}-kernel ./dist/vmlinuz-${ARCH}
-	rm -rf out
-	cd ./dist && tar -czvf ../hook-${GIT_VERSION}.tar.gz ./*
-
-deploy: dist
-ifeq ($(shell git rev-parse --abbrev-ref HEAD),main)
-	s3cmd sync ./hook-${GIT_VERSION}.tar.gz s3://s.gianarb.it/hook/${GIT_VERSION}.tar.gz
-	s3cmd cp s3://s.gianarb.it/hook/hook-${GIT_VERSION}.tar.gz s3://s.gianarb.it/hook/hook-main.tar.gz
-endif
-
-.PHONY: clean
-clean:
-	rm ./hook-${GIT_VERSION}.tar.gz
-	rm -rf dist/ out/ tink-docker/local/ bootkit/local/
+help: ## Print this help
+	@grep --no-filename -E '^[[:alnum:]_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sed 's/:.*## /·/' | sort | column -t -W 2 -s '·' -c $$(tput cols)
```

Comment:
Everywhere I tried to use this really didn't like that special character. Replacing it with .e.g. `|` fixed things for me.

Reply:
Hmm we use this in all the other Makefiles too (in boots, tink, tinkerbell-docs). I've been copy/pasting this from when I first got this snipped and I think the idea is to have a character that is not likely to be present in the comment string. I'm curious if you have a non UTF8 locale set?